src/tiffimage_int.cpp

```diff
@@ -1830,8 +1830,8 @@ void TiffCreator::getPath(TiffPath& tiffPath, uint32_t extendedTag, IfdId group,
     tiffPath.push(TiffPathItem(extendedTag, group));
```

Comment:
Oh this is funny. clang-tidy should but is not recommending emplace() here.

Reply:
fixed